### PR TITLE
PMM-7 fix pmm2-client docker image issue

### PR DESCRIPTION
**Pull request by nikita-b**
We don't have `find` package in our image

https://jira.percona.com/browse/PMM-0

- [ ] Links to other linked pull requests.

### CODE REVIEW

**Review by ademidoff — APPROVED** [review]

**Review by ademidoff — APPROVED** [review]

File: build/bin/install_tarball [hunk 1]
```diff
@@ -42,13 +42,13 @@ ${INSTALL_COMMAND} -m 0755 queries-postgres-uptime.yml "${PMM_DIR}"/collectors/c
 ${INSTALL_COMMAND} -m 0755 queries.yaml "${PMM_DIR}"/collectors/custom-queries/postgresql/medium-resolution
 ${INSTALL_COMMAND} -m 0755 queries-mysqld-group-replication.yml "${PMM_DIR}"/collectors/custom-queries/mysql/high-resolution
 
-find ./bin -maxdepth 1 -type f | while read -r FILE; do
+for FILE in $( ls ${PWD}/bin ); do
```
Comment (cezmunsta) [inline build/bin/install_tarball:45]: This is not clean code and will fail lint test.
Also, IMO this is the wrong way to address the problem. The problem is that the base image for the container doesn't have find, but that is trivial to fix by using a different base image and copy files in with `COPY - - from`

**Review by cezmunsta — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2656-7199fd0
client docker - perconalab/pmm-client-fb:PR-2656-7199fd0
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2656-7199fd0.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2656-7199fd0&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2656-7199fd0.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10709/

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/10254/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/19423/

**Pull request merged**